include/benchmark/benchmark.h

```diff
@@ -373,11 +373,20 @@ class Counter {
     kAvgIterationsRate = kIsRate | kAvgIterations
   };
 
+  enum One_K {
```

Comment:
nit: https://google.github.io/styleguide/cppguide.html#Enumerator_Names

OneK is a better name.